src/Microsoft.Extensions.Logging.EventLog/Internal/IEventLog.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Diagnostics;
+
+namespace Microsoft.Extensions.Logging.EventLog.Internal
+{
+    public interface IEventLog
+    {
+        int MaxMessageSize { get; }
```

Comment:
Why have a property for this? Might be easier to hard-code it in the calling class, and the unit tests can use the real number. (Yeah it means huge strings, but is that a real problem?)